packages/build/src/steps/run_step.ts

```diff
@@ -67,16 +67,17 @@ export const runStep = async function ({
   // Add relevant attributes to the upcoming span context
   const attributes: StepExecutionAttributes = {
     'build.execution.step.name': coreStepName,
-    'build.execution.step.description': coreStepDescription,
     'build.execution.step.package_name': packageName,
     'build.execution.step.id': coreStepId,
     'build.execution.step.loaded_from': loadedFrom,
     'build.execution.step.origin': origin,
     'build.execution.step.event': event,
   }
   const spanCtx = setMultiSpanAttributes(attributes)
+  // If there's no `coreStepId` then this is a plugin execution
+  const spanName = `run-step-${coreStepId || 'plugin'}`
```

Comment:
`coreStepId` only exists for internal steps. We replace this for plugin for plugin executions and we can use the `package_name` to aggregate on the package.

Reply:
do you mind giving more information about the plugin vs steps?

For example in [this comment](https://github.com/netlify/build/blob/1d66a80185f820e3ad463a63134719f1b22052ee/packages/build/src/steps/run_step.ts#L16), it mentions we have core steps, build commands or plugin. I understand the meaning of the steps we do in buildbot, but not in here. Maybe this is a question too broad, so we can catch up on "Making Moves"